chapter_appendix_math/single-variable-calculus_vn.md

```diff
@@ -4,47 +4,52 @@
 # Single Variable Calculus
 -->
 
-# *dịch tiêu đề phía trên*
+# Giải tích đơn biến
```

Comment:
Mình nhớ có nhiều sách dùng "giải tích một biến" và "giải tích nhiều biến".

Reply:
Em googgle thử thì thấy toàn xài `một biến` `nhiều biến` thật.
```suggestion
# Giải tích một biến
```